app/common/utils.js

```diff
@@ -0,0 +1,7 @@
+export function shiftItemRandomly(array) {
+    return array.splice(getRandomInt(0, array.length), 1)[0];
```

Comment:
Since shift and splice take and remove an item from an array, I thought both names would be illustrative. This methods takes an item randomly from an array, slplicing randomly a length-1 array. I guess that name, "takeItemRandomlyFromArray", would be better, isn't it?